src/rdbms/rdbms_queries.erl

```diff
@@ -823,20 +828,54 @@ create_bulk_insert_query(Table, Fields, RowsNum) when RowsNum > 0 ->
         -> {SQL :: nonempty_string(), []} | {[], MSSQL::nonempty_string()}.
 get_db_specific_limits(Limit) ->
     LimitStr = integer_to_list(Limit),
-    do_get_db_specific_limits(?RDBMS_TYPE, LimitStr).
+    do_get_db_specific_limits(?RDBMS_TYPE, LimitStr, false).
 
 -spec get_db_specific_offset(integer(), integer()) -> iolist().
 get_db_specific_offset(Offset, Limit) ->
     do_get_db_specific_offset(?RDBMS_TYPE, integer_to_list(Offset), integer_to_list(Limit)).
 
 
-do_get_db_specific_limits(mssql, LimitStr) ->
+do_get_db_specific_limits(mssql, LimitStr, false) ->
```

Comment:
We definitely need a spec or a variable name here to understand what this 'false' and 'true' means.